pnpm-lock.yaml

```diff
@@ -3179,8 +3179,6 @@ importers:
 
   packages/cli/cli/dist/dev: {}
 
```

Comment:
is this an expected change?

Reply:
weird, i dont think so but it shouldn't matter